autoPyTorch/evaluation/train_evaluator.py

```diff
@@ -34,12 +34,74 @@ def _get_y_array(y: np.ndarray, task_type: int) -> np.ndarray:
 
 
 class TrainEvaluator(AbstractEvaluator):
+    """
```

Comment:
Almost same as another documentation?

Reply:
Yes, because it is an implementation of the abstract class. How will you change it? Or do you see an issue with this?